From ffd97ac5932d322ad0ea1c4535459fba4f703f72 Mon Sep 17 00:00:00 2001
From: Christopher Degawa <ccom@randomderp.com>
Date: Sat, 23 Apr 2022 16:31:31 -0500
Subject: [PATCH 2/2] jxl: limit not-renaming to msvc

mingw-w64 relies on the library being jxl for linking with ffmpeg

Signed-off-by: Christopher Degawa <ccom@randomderp.com>
---
 lib/jxl.cmake         | 2 +-
 lib/jxl_threads.cmake | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/lib/jxl.cmake b/lib/jxl.cmake
index 1d44dc6..ee762db 100644
--- a/lib/jxl.cmake
+++ b/lib/jxl.cmake
@@ -530,7 +530,7 @@ endif()  # JPEGXL_ENABLE_TCMALLOC
 
 # Install the static library too, but as jxl.a file without the -static except
 # in Windows.
-if (NOT WIN32)
+if (NOT MSVC)
   set_target_properties(jxl-static PROPERTIES OUTPUT_NAME "jxl")
   set_target_properties(jxl_dec-static PROPERTIES OUTPUT_NAME "jxl_dec")
 endif()
diff --git a/lib/jxl_threads.cmake b/lib/jxl_threads.cmake
index d0d2d00..a49cb37 100644
--- a/lib/jxl_threads.cmake
+++ b/lib/jxl_threads.cmake
@@ -40,7 +40,7 @@ set_target_properties(${_target} PROPERTIES
 
 # Always install the library as jxl_threads.{a,so} file without the "-static"
 # suffix, except in Windows.
-if (NOT WIN32)
+if (NOT MSVC)
   set_target_properties(${_target} PROPERTIES OUTPUT_NAME "jxl_threads")
 endif()
 install(TARGETS ${_target}
-- 
2.35.1

